docs/blog/adafruit-circuit-playground-express/v0.20.13.md

```diff
@@ -0,0 +1,75 @@
+# Adafruit Circuit Playground Express Editore v0.20.13
+
+**Posted on November 13, 2017 by [pelikhan](https://github.com/pelikhan)**
+
+We are very excited to remove the ``beta`` label from https://makecode.adafruit.com .
+This update brings **Scratch** style rendering of the blocks, better support for assistive technologies, a console view and a [many](https://github.com/Microsoft/pxt/commits/master) [many](https://github.com/Microsoft/pxt-common-packages/commits/master) [many](https://github.com/Microsoft/pxt-adafruit/commits/master) fixes.
+
+
+## Beta -> Stable
+
+The editor is now out of ``beta`` into a stable state. You should not expect any API change or major UI change until the next school calendar year.
```

Comment:
which school calendar year? The US?

Reply:
API change -> API changes
UI change -> UI changes